src/main/kotlin/io/snyk/plugin/services/download/CliDownloaderService.kt

```diff
@@ -109,25 +99,34 @@ class SnykCliDownloaderService {
         }
     }
 
+
+    // no cli -> download (currentVersion = 0 -> != 10
```

Comment:
Do we mean to keep these comments in here?

Reply:
Could be helpful, but maybe better as a JavaDoc :)